.gitignore

```diff
@@ -14,5 +14,6 @@ log/*
 .yardoc
 doc/
 tmp
+Gemfile.lock
```

Comment:
I normally include lockfiles in version control, also for gems, to have a more consistent development experience. But in this case, the main Gemfiles in `gemfiles/` folder were already gitignore so my practice was not even being followed so I'm cool with gitignoring all lockfiles.

Regarding security scanner warnings, I think the issue there was including the lockfile in the packaged gem, not including the lockfile in source control. But stopping including it in source control also fixes the issue as a side effect 👍.